packages/editor-ui/src/components/Node.vue

```diff
@@ -34,7 +34,7 @@
 		</div>
 		<div class="node-description">
 			<div class="node-name" :title="data.name">
-				{{data.name}}
+        {{translateSpecific({key: `${data.type}.displayName`, fallback: data.name})}}
```

Comment:
For readability and consistency, `translateSpecific` is meant to be called only internally by the named methods that are used in the templates, i.e. `translateSpecific` is not to be used directly, as specified in its docstring. Ideally we should reuse one of the `$` methods in `translate.ts` or create one if none fit.

Reply:
Fixed, thanks!